.circleci/config.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         paths:
         - /go/pkg/mod
     - run: go test -race -cover ./...
+    - run: find ./sasl -name go.mod -execdir go test -race -cover ./... \;
```

Comment:
I know this was my idea, but it looks like it was a bad one, sorry about that! It looks like the exit code from the `-execdir` is being lost, so CircleCI is treating these as passing even though they panic. :(

If we don't have any other ideas, feel free to revert back to what you had before.